pycbc/filter/resample.py

```diff
@@ -178,6 +178,34 @@ def resample_to_delta_t(timeseries, delta_t, method='butterworth'):
 _highpass_func = {numpy.dtype('float32'): lal.HighPassREAL4TimeSeries,
                  numpy.dtype('float64'): lal.HighPassREAL8TimeSeries}
 
+def notch_fir(timeseries, f1, f2, order, beta=5.0):
+    """ notch filter the time series using an FIR filtered generated from
+    the ideal response passed through a kaiser window (beta = 5.0)
```

Comment:
prefer to say *time domain* kaiser window (if this is what the scipy function does)